opentelemetry-prometheus/Cargo.toml

```diff
@@ -21,17 +21,20 @@ rustdoc-args = ["--cfg", "docsrs"]
 
 [dependencies]
 once_cell = { workspace = true }
-opentelemetry = { version = "0.26", default-features = false, features = ["metrics"] }
-opentelemetry_sdk = { version = "0.26", default-features = false, features = ["metrics"] }
+opentelemetry = { version = "0.27", path = "../opentelemetry", default-features = false, features = ["metrics"] }
```

Comment:
please remove the path="..". It is intentional until we hit 1.0

Reply:
ok, removed :)